src/menus/TrackerMenu/TrackerMenu.js

```diff
@@ -43,33 +45,18 @@ class TrackerMenu extends Component {
   }
 
   componentDidMount() {
-    const { layerService } = this.props;
-
-    this.trackerLayers = layerService
-      .getLayersAsFlatArray()
-      .filter((l) => l.isTrackerLayer);
-
-    unByKey(this.olEventsKeys);
-    this.olEventsKeys = [];
-    if (this.trackerLayers.length) {
```

Comment:
Is this if clause needed?

Reply:
I removed it since it ins't required on a forEach loop